Yohanan-Negash/miniworkshop
# Bootstrap NEXT with tailwind

Resolves: #4 

## Solution 
1. Setup NEXTJS by using the following command : 
``` js 
 yarn create next-app --typescript
```
2. Setup Tailwind with NEXT by following this [link](https://tailwindcss.com/docs/guides/nextjs).
## Testing 
1. Start server ; `yarn dev`
2. Navigate to the Home Page ; `/` and observe the 'Hello world'  text at the top of the page.
